<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurvatureDialog</class>
 <widget class="QDialog" name="CurvatureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Horizon Curvature</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbInput">
       <property name="text">
        <string>Horizon:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbInput">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Base Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leBaseName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Curvature Attributes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cbKMean">
        <property name="text">
         <string>Mean</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbKGauss">
        <property name="text">
         <string>Gaussian</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbKMin">
        <property name="text">
         <string>Minimum</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbKMax">
        <property name="text">
         <string>Maximum</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbKMaxNeg">
        <property name="text">
         <string>Maximum Negative</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbKMaxPos">
        <property name="text">
         <string>Maximum Positive</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbDip">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Dip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cStrike">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Strike</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbDipAngle">
        <property name="text">
         <string>Dip Angle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbDipAzimuth">
        <property name="text">
         <string>Dip Azimuth</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CurvatureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CurvatureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
